WordPress/openverse
# Add favicon to Storybook

## Description



This PR adds a favicon, the pink one, to Storybook.

<img width="338" alt="Screenshot 2024-06-05 at 7 46 57 PM" src="https://github.com/WordPress/openverse/assets/16580576/ff1d4138-f852-4255-a1e1-c07913b950ba">

## Testing instructions

0. Check out this PR.
1. Run Storybook `just frontend/run storybook`, see the favicon at `https://localhost:54000`.
2. Build Storybook `just frontend/run storybook:build-for-docs`, open the built folder `cd frontend/storybook-static`, run the server `python3 -m http.server 54000`, see the favicon at `https://localhost:54000`.

## Checklist


      PRs) or the media properties generator (`just catalog/generate-docs media-props`
      for the catalog or `just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
